CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+=== 0.2.1 (unreleased) ===
+- Changed Notification.notification_class and Notification.verbose_name fields to have less restrictive `max_length`
```

Comment:
@jordanmkoncz we actually haven't released 0.2 yet, so would you mind just moving this down into the 0.2 notes?